README.md

```diff
@@ -205,6 +191,20 @@ const App = () => {
 ReactDOM.render(<App />, node); // "Alex" will be rendered
 ```
 
+### Templates
```

Comment:
Nice, I think this place is better to describe hooks limitation after its documentation 👍